app/cdap/components/GridTable/components/MenuItemComponent/types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export interface IMenuItem {
+  label?: string;
+  supportedDataType?: string[];
+  value?: string;
```

Comment:
Is `value` only used for `'heading'` and `'divider'`? Can those strings be typed here?

Reply:
So it's `heading` or `divider` or the name of the transformation?

Reply:
It is all the three, at few places in menu we need heading and a divider So we are basically checking if value is 'heading' or is 'divider' we will display label as heading and a normal div for divider, else they will be treated as menu items